README.md

```diff
@@ -3,202 +3,212 @@
 [ci-badge]: https://travis-ci.org/firebase/firebase-jobdispatcher-android.svg?branch=master
 [ci-link]: https://travis-ci.org/firebase/firebase-jobdispatcher-android
 
-The Firebase Android JobDispatcher is a library that provides a high-level wrapper around job
-scheduling engines on Android, starting with the [GCM Network Manager][nts].
+The Firebase JobDispatcher is a library for working with the [JobScheduler][] compatibility library
```

Comment:
s/ the [JobScheduler][] compatibility library ... / This library uses the services provided by the Google Play services application to extend the [Android JobScheduler APIs] to android versions before Lollipop.